### DISCUSSION TITLE
Angles in the dataset

### DISCUSSION THREAD

**Discussion by santiagolarregla** [PaddlePaddle/PaddleSpatial#83] [BaiduKDDCup2022] [answered]
Hi,  would it be possible to have a picture showing the meaning of the variables Wdir and Ndir, I don't fully understand the values of these variables. 
Ndir is defined as the yaw angle of the nacelle, but with respect to what?

**Comment by ramber1836** [comment, marked as answer]
Hello there,

You can refer to this report (https://www.mdpi.com/1996-1073/11/3/553/htm)
for the angle explanation, and an illustrative pic is also provided:

[image: image.png]

Thanks.

Regards,
Xinjiang Lu

**Comment by ramber1836** [comment]
Regards,
Xinjiang Lu
